src/components/managerlib/pdfwatcher.ts

```diff
@@ -7,10 +7,18 @@ export class PdfWatcher {
     private readonly extension: Extension
     private readonly pdfsWatched = new Set<string>()
     private readonly pdfWatcher: chokidar.FSWatcher
+    readonly vscodeFsWatcher: vscode.FileSystemWatcher
+    private readonly pdfUrisWatched = new Set<string>()
```

Comment:
It seems that this property is never read. We just add elements to it in `watchPdfFile`.
Am I missing something?